src/atwho.js

```diff
@@ -17,20 +17,21 @@ define([
 
     // stripped down version of http://codepen.io/ImagineProgramming/storydump/javascript-memoization-timeout
     var timed = function timed(f, timeout) {
-        var cache, time = 0;
+        var cache = {}, time = 0;
```

Comment:
This is the kind of module-level global that will leak between separate instances of vellum on the same page. The leak could be avoided by saving these as properties of `this.data.atwho` in [`init()`](https://github.com/dimagi/Vellum/blob/01155fa227dddfaadea348d72c511bfff99df107/src/uploader.js#L268) or [`loadData()`](https://github.com/dimagi/Vellum/blob/01155fa227dddfaadea348d72c511bfff99df107/src/lock.js#L25).